book/website/collaboration/ambassador-schemes/ambassador-schemes-establish.md

```diff
@@ -0,0 +1,85 @@
+(cl-ambassador-schemes-establish)=
+# Establishing and Running an Ambassador Scheme 
+
+(cl-ambassador-schemes-establish-setup)=
+
+## Setting up the Scheme
+
+To launch an ambassador scheme, you will likely need to work with and get approval and sign-off from different departments in your organisation, such as finance, legal, communications and project management.
+
+This may require thinking about how the scheme will contribute to organisational priorities such as capacity building, wider engagement of research and/or training so you can secure leadership buy-in. 
```

Comment:
I feel like there are a lot of lists back-to-back/ in a short span of writing. Not necessarily an issue, just thought to bring it up.